### PR TITLE
Update repo with cookiecutter

### PR DESCRIPTION
**Pull request by varsha888**
[Issue](https://github.com/mobilecoinfoundation/build-rs/issues/16)

Update repo with changes from [cookiecutter](https://github.com/mobilecoinfoundation/cookiecutters)

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+repository:
+  name: "sgx-sigstruct"
+  description: "A brief summary of the workspace"
```
Comment (nick-mobilecoin) [inline .github/settings.yml]: :wrench: 
```suggestion
  description: "Enclave Signature Structure (SIGSTRUCT)"
```

File: css/Cargo.toml [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
 name = "mc-sgx-css"
 version = "0.1.0"
-authors = ["MobileCoin"]
+authors = { workspace = true }
 # See https://crates.io/category_slugs for valid categories
-categories = ["development-tools", "encoding"]
-description = "Enclave signature struct extractor"
-edition = "2021"
+categories = ["[development-tools", "encoding]"]
+description = "A brief summary of the crate"
```
Comment (nick-mobilecoin) [inline css/Cargo.toml]: :wrench:
```suggestion
description = "Enclave signature struct extractor"
```

**Review by nick-mobilecoin — APPROVED** [review]
Looks good, some minor fixes.

I missed it in https://github.com/mobilecoinfoundation/build-rs/pull/19 as well, was this done with cruft? If so the cruft.json file should be part of the commit so that future updates have a baseline.

**Comment by varsha888** [comment]
> Looks good, some minor fixes.
> 
> I missed it in [mobilecoinfoundation/build-rs#19](https://github.com/mobilecoinfoundation/build-rs/pull/19) as well, was this done with cruft? If so the cruft.json file should be part of the commit so that future updates have a baseline.

Not with cruft, the diff was too large and cruft conflict resolution was painful

**Review by awygle — APPROVED** [review]

**Pull request merged**